react/components/Payment/PaymentMethod.tsx

```diff
@@ -61,37 +59,32 @@ class PaymentMethod extends Component<Props & InjectedIntlProps> {
               }
             )}`}
           </p>
-          {isBankInvoice && payment.url && (
-            <ButtonLink url={payment.url} variation="primary" openNewWindow>
-              {intl.formatMessage(
-                { id: 'payments.bankinvoice.print' },
-                { paymentSystemName: payment.paymentSystemName }
-              )}
-            </ButtonLink>
-          )}
-          <div hidden={!open}>
-            <p className="c-muted-2">
-              {intl.formatMessage({ id: 'payments.id' }, { id: payment.id })}
-            </p>
-            <p className="c-muted-2">
-              {intl.formatMessage(
-                { id: 'payments.transaction.id' },
-                {
-                  id: transactionId,
-                }
-              )}
-            </p>
+          <div
+            className="ml4"
+            onMouseEnter={() => setIsOpen(true)}
```

Comment:
Suggestion: 
```
onMouseEnter={this.setIsOpen}
onMouseLeave={this.setIsClosed}
```

Reply:
@kevinch 	that's how you do it now in a function components using Hooks.

Check this: https://reactjs.org/docs/hooks-faq.html#are-hooks-slow-because-of-creating-functions-in-render

Reply:
Yep, hoooooks 😍